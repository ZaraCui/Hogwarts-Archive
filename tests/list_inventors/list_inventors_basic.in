ADD COLLECTION tests/data/books_small.csv
LIST INVENTORS
EXIT

