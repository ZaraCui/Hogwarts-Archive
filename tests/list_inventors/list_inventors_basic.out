user: 12 spellbooks successfully added.

user: Filius Flitwick
Godric Gryffindor
Helga Hufflepuff
Libatius Borage
Minerva McGonagall
Newt Scamander
Salazar Slytherin

user: Ending Archive process.
